source/crud/crud.rst

```diff
@@ -306,6 +306,18 @@ Read
      * @see http://docs.mongodb.com/manual/reference/command/aggregate/
      */
     hint: Optional<(String | Document)>;
+
+    /**
+     * Map of parameter names and values. Values must be constant or closed
+     * expressions that do not reference document fields. Parameters can then be
+     * accessed as variables in an aggregate expression context (e.g. "$$var").
+     *
+     * This option is sent only if the caller explicitly provides a value. The default is to not send a value.
+     * This option is only supported by servers >= 5.0. Older servers >= 2.6 (and possibly earlier) will report an error for using this option.
```

Comment:
I tested as far back at 2.6 via the `mongo` shell (since libmongoc/PHPC only talk to 3.0+). Since I could verify before then, the corresponding server-side error test requires 2.6+. That should be fine since I don't think any modern drivers communicate with earlier servers anyway.